Specify the encoding of the source files. Otherwise javac chokes on some special symbol.
author: Masao Fujinaga (University of Alberta, Compute Canada)
--- Butterfly/src/build.xml.orig	2017-03-08 19:26:27.231444367 +0000
+++ Butterfly/src/build.xml	2017-03-08 19:27:19.564945303 +0000
@@ -42,6 +42,7 @@
         <javac debug="true" debuglevel="${debuglevel}" destdir="bin" source="${source}" target="${target}">
             <src path="src"/>
             <classpath refid="TrasncriptomeAssembly.classpath"/>
+            <compilerarg line="-encoding UTF8" />
         </javac>
     </target>
     <target description="Build all projects which reference this project. Useful to propagate changes." name="build-refprojects"/>
